docker-compose.nginx.yml

```diff
@@ -29,7 +29,8 @@ services:
       - ${HTTP_PORT}:80
   mattermost:
     ports:
-      - ${CALLS_PORT}:8443/udp
+      - ${CALLS_PORT}:${CALLS_PORT}/udp
+      - ${CALLS_PORT}:${CALLS_PORT}/tcp
```

Comment:
One qq for my own learning: if you don't set this var, what does docker-compose do?

Reply:
@cpoile There is a default so one would have to explicitly unset it. But if you did then you'd get a bunch of warnings and eventually a failure:

```
WARNING: The CALLS_PORT variable is not set. Defaulting to a blank string.
ERROR: The Compose file './docker-compose.nginx.yml' is invalid because:
services.mattermost.ports contains an invalid type, it should be a number, or an object
services.mattermost.ports contains an invalid type, it should be a number, or an object
```